tests/wicked/basic/ref/t08_setup_second_card.pm

```diff
@@ -22,6 +22,7 @@ use lockapi;
 sub run {
     my ($self, $ctx) = @_;
     record_info('Info', 'Set up a second card');
+    assert_script_run(sprintf("ip a a %s/15 dev %s", $self->get_ip(type => 'second_card'), $ctx->iface()));
```

Comment:
I'm not sure, but I guess we do not use wicked on the REF at all? If not, I would prefer, that we do not mix our configuration.

Reply:
this is needed in REF only in this test because SUT will try to ping 10.0.3.0/24 and it must be set in REF beforehand.

Reply:
yeah, but we always have a working IP configuration on REF and my question is: "Where does it came from?"

Reply:
it comes from before_tests.pm, I didn't call `reset_wicked()` in REF.